static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -257,7 +257,7 @@ export function SearchQueryBuilder({...props}: SearchQueryBuilderProps) {
 }
 
 const Wrapper = styled(Input.withComponent('div'))`
-  min-height: 38px;
+  min-height: ${p => p.theme.form.md.minHeight};
```

Comment:
this has no impact on the old theme, as 38px is the md minHeight there, but it’s different for chonk